src/test/java/org/mitre/synthea/world/agents/PayerTest.java

```diff
@@ -558,6 +561,8 @@ public void personCannotAffordPayer() {
   @SuppressWarnings("unchecked")
   public void payerMemberYears() {
     long startTime = Utilities.convertCalendarYearsToTime(2000);
+    Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+    calendar.setTimeInMillis(startTime);
```

Comment:
Did you mean to do something with this `Calendar`?

Reply:
I think this is a rebase artifact, I had independently fixed this test when it broke. I'll delete those two lines.